src/ffjord.jl

```diff
@@ -157,8 +157,9 @@ function jacobian_fn(f, x::AbstractMatrix)
     Zygote.@ignore fill!(z, zero(eltype(x)))
     vec = Zygote.Buffer(x, size(x, 1), size(x, 1), size(x, 2))
     for i in 1:size(y, 1)
-        Zygote.@ignore z[i, :] .+= one(eltype(x))
-        vec[i, :, :] = back(z)[1]
+        z_ = copy(z)
```

Comment:
it would be better to reset to zero I think.

Reply:
Yes, I used `BenchmarkTools`, the results are:
```julia
# copy
  38.800 μs (291 allocations: 10.91 KiB)
# reset
  37.200 μs (291 allocations: 10.91 KiB)
```